lib/charms/istio_beacon_k8s/v0/service_mesh.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""#Service Mesh Library.
+
+The service mesh library is used to facilitate adding you charmed application to a service mesh.
+The library leverages the `service_mesh` interface and the `cross_model_mesh` interface
+
```

Comment:
iiuc the `cross_model_mesh` isn't needed anymore, but if I'm wrong reject this.

```suggestion
The service mesh library is used to facilitate adding your charmed application to a service mesh.
The library leverages the `service_mesh` interface.

```

Reply:
Yeah we still need cross_model_mesh for sure.